services/gateway/sources/version2/newsfeed.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const newsfeed = require('./mappings/newsfeed');
+
+module.exports = {
+	type: 'moleculer',
+	method: 'newsfeed.articles',
+	params: {
+		source: '=,string',
+		offset: '=,number',
+		limit: '=,number',
+	},
+	definition: {
+		data: ['data', newsfeed],
+		meta: {
+			count: '=,number',
```

Comment:
Same as above.

Reply:
Good catch. In the version 2 the response was missing, but it wouldn't hurt to add it. I've updated the specs, can you please add it?

Reply:
Updated version 2 meta mappings.